packages/teams-js/src/public/interfaces.ts

```diff
@@ -1053,3 +1053,23 @@ export interface AdaptiveCardVersion {
   /** Represents the minor version number. */
   minorVersion: number;
 }
+
+/**
+ * Currently supported Mime type
+ */
+export enum SupportedMimeType {
```

Comment:
Since this is for clipboard, might make sense to change the name to something like ClipboardSupportedMimeType?

Reply:
Sure, will update it as `ClipboardSupportedMimeType`